packages/calcite-components/src/utils/form.tsx

```diff
@@ -1,6 +1,17 @@
 import { closestElementCrossShadowBoundary, queryElementRoots } from "./dom";
 import { FunctionalComponent, h } from "@stencil/core";
 
+/**
+ * Any form <Component> with a `calcite<Component>Input` event needs to be included in this array.
+ * Exported for testing purposes.
+ */
+export const componentsWithInputEvent = [
+  "calcite-input",
+  "calcite-input-number",
+  "calcite-input-text",
+  "calcite-text-area",
+];
+
```

Comment:
We need to make sure to update this array if additional Input components are added

Reply:
Interesting, could you elaborate on this?